2020/join/gatsby-new-join/src/components/business.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from "react"
```

Comment:
single quotation mark is preferred over double quotation mark. So change this line to `import React, { useState } from 'react'`